hera_cal/lstbin.py

```diff
@@ -579,110 +577,137 @@ def lst_bin_files(data_files, filetype='uvh5', input_cals=None, dlst=None, verbo
     kwargs['outdir'] = outdir
     kwargs['overwrite'] = overwrite
 
-    # get metadata from the zeroth data file
-    d, fl, ap, a, f, t, l, p = io.load_vis(data_files[0][0], return_meta=True, pick_data_ants=False, filetype=filetype)
+    # get metadata from the zeroth data file in the last day
+    uv = UVData()
```

Comment:
All of this is much more easily done with HERAData. Most of this copies what's already stored in the metadata of a HERAData object (freqs, antpos, ants, times, etc.)